tests/Utils/Policies/UserPolicy.php

```diff
@@ -2,17 +2,28 @@
 
 namespace Tests\Utils\Policies;
 
+use Illuminate\Auth\Access\Response;
 use Tests\Utils\Models\User;
 
 class UserPolicy
 {
+    public const SUPER_ADMIN = 'super admin';
     public const ADMIN = 'admin';
 
     public function adminOnly(User $user): bool
     {
         return $user->name === self::ADMIN;
     }
 
+    public function superAdminOnly(User $user): Response
+    {
+        if ($user->name === self::SUPER_ADMIN) {
+            return Response::allow();
+        }
+
+        return Response::deny('Only super admins allowed');
```

Comment:
That can go in a constant and be referenced from the test to make the connection clear.

Reply:
Done.